dynamics/applications/CMakeLists.txt

```diff
@@ -3,8 +3,51 @@ set(DynamicsSrc "../src")
 set(SRC_DIR "${DynamicsSrc}")
 set(INCLUDE_DIR "${DynamicsSrc}/include")
 
+set(CoreDir "../../core/src/")
+
 include_directories(${INCLUDE_DIR})
 
+add_executable(createNextsimMeshData createNextsimMeshData.cpp
+    "${CoreDir}/modules/ParametricGrid.cpp"
```

Comment:
Comment: As before with test dependencies. Might be worth including more than we need for simplicity (I know, I know but would be cool to just include core, physics, dynamics as needed --- I've been looking at this recently which is the only reason I use the word 'cool'). Obvs not every test will want the bulked out set but could save a lot of code & time & lines. 

We also currently make everything, including .o files, for each test in it's own sub directory but in principle could we share them if build time is a concern (currently the entirety of the testing time, less a second).

Reply:
My general attitude with CMake, like with catch2, is to get something to work, and refactor once that is happening. Also, my CMake experience is currently a bit thin, so it would require a bit of learn & play time for me to do anything much more advanced.